### PR TITLE
Add AZBA

### PR DESCRIPTION
**Pull request by kailynkfields**

### CODE REVIEW

File: bg_atlasgen/atlas_scripts/azba_zfish.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    ATLAS_LINK = "http://www.azba.wayne.edu"
+    CITATION = "Kenney et al. 2021, https://doi.org/10.1101/2021.05.04.442625"
+    ATLAS_FILE_URL = "http://www.azba.wayne.edu/2021-08-22_AZBA.tar.gz"
+    ORIENTATION = "las"
+    ROOT_ID = 9999 
+    ATLAS_PACKAGER = "Kailyn Fields, kailyn.fields@wayne.edu"
+    ADDITIONAL_METADATA = {}
+    
+    
+    #setup folder for downloading
```
Comment (adamltyson) [inline bg_atlasgen/atlas_scripts/azba_zfish.py:43]: To ensure that >1 atlas can be created at the same time I'd add this line here:
```python
working_dir = working_dir / "brainglobe_workingdir" / ATLAS_NAME
```

**Comment by adamltyson** [comment]
This looks great @kailynkfields, I left one tiny comment, then I'm happy to merge.

@FedeClaudi has already ok'd this. @vigji could you take a look? Zfish are more your thing than mine.

File: bg_atlasgen/atlas_scripts/azba_zfish.py (hunk 1 above)
Comment (kailynkfields) [reply bg_atlasgen/atlas_scripts/azba_zfish.py:43]: Even though working_dir is already working_dir / "brainglobe_workingdir" ?

Comment (adamltyson) [reply bg_atlasgen/atlas_scripts/azba_zfish.py:43]: Ah good catch, sorry it should be:
```python
working_dir = working_dir  / ATLAS_NAME
```

**Comment by adamltyson** [comment]
Thanks @kailynkfields, I think this is ready to merge once @vigji is happy.

**Comment by vigji** [comment]
yes, good to go for me! Thank you very much for your contribution @kailynkfields!

**Comment by adamltyson** [comment]
:tada: Thanks @kailynkfields!

**Pull request merged**